canonical/global-nav
# WD-2373 Implement new mobile side navigation

## Done

- Implement the new visual design for mobile navigation

## QA

- Check out the demo [here](https://global-nav-256.demos.haus/).
- Compare the menu implementation to the [visual design](https://www.figma.com/file/BSP7FqnRMcMvO7kCozJKUa/Ubuntu.com-navigation-update-Iteration-9?node-id=1323%3A3714&t=qlM88womaW4j6D5q-0).

## Issue / Card

- [WD-2373](https://warthogs.atlassian.net/browse/WD-2373)


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)


[WD-2373]: https://warthogs.atlassian.net/browse/WD-2373?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
